pom.xml

```diff
@@ -142,13 +142,18 @@
             </dependency>
             <dependency>
                 <groupId>org.bouncycastle</groupId>
-                <artifactId>bcmail-jdk15on</artifactId>
-                <version>1.70</version>  <!--e2e tests and samples only-->
+                <artifactId>bcmail-jdk18on</artifactId>
```

Comment:
jdk15on versions are no longer maintained, so we need to move to jdk18on to stay up to date on security patches